.github/workflows/amd-mi200.yml

```diff
@@ -14,7 +18,7 @@ permissions:
   issues: write
 
 jobs:
-  amd-tests:
+  unit-tests:
```

Comment:
FYI @mrwyattii - this would make it required now when modifying the files in the path list above

Reply:
Nevermind